www/css/app.css

```diff
@@ -260,11 +298,11 @@ button {
     .btn-outline-secondary {
         font-size: 10px !important;
     }
-
     /* Reduce size of the bottom buttons */
-    .btn-lg, .btn-group-lg > .btn {
+    .btn-lg,
+    .btn-group-lg>.btn {
         padding-top: 1px !important;
         padding-bottom: 1px !important;
         font-size: 16px !important;
     }
-}
+}
```

Comment:
@Hardik500 You seem to have done a whole load of style changes in CSS which have nothing to do with this PR! I suggest you revert all style changes on this file, and if the file hasn't changed in any meaningful way, don't include it in the PR.